server/README.rst

```diff
@@ -341,3 +341,28 @@ This will find jobs tagged with both "foo" and "bar".
   .. code-block:: console
 
     $ curl -X GET http://localhost:8000/v1/agents/data
+
+**[POST] /v1/status - Receive job status updates from an agent and posts them to the specified webhook.
```

Comment:
See my comment about the path for this API in v1.py, and make sure the README is consistent with what we have there.